DNN Platform/DotNetNuke.Abstractions/Portals/IPortalAliasInfo.cs

```diff
@@ -1,27 +1,54 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+
 namespace DotNetNuke.Abstractions.Portals
 {
-    using System.Data;
-    using System.Xml;
-    using System.Xml.Schema;
+    using DotNetNuke.Abstractions.Urls;
 
+    /// <summary>
+    /// Portal Alias Info interface.
+    /// </summary>
     public interface IPortalAliasInfo
     {
-        //BrowserTypes BrowserType { get; set; }
-        string CultureCode { get; set; }
+        /// <summary>
+        /// Gets or sets the http alias.
+        /// </summary>
         string HTTPAlias { get; set; }
-        bool IsPrimary { get; set; }
-        int KeyID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal Alias ID.
+        /// </summary>
         int PortalAliasID { get; set; }
```

Comment:
@mitchelsellers what do you think about enforcing a consistent naming convention for `Id` over `ID` in these new interfaces?

```suggestion
        int PortalAliasId { get; set; }
```

Reply:
I guess this has already been released, so this would end up being a breaking change 😞

Reply:
If we can't make an exception, I can strategically deprecate all the `ID` suffixes for `Id` and configure everything to flow through the `Id` value. Kind of a pain to implement and maintain, but we should do our best to follow our deprecation rules 😟

Reply:
@ahoefling That would be great, but otherwise I think we make note for 10.0.0

Reply:
:+1: for adding `PortalAliasId` and deprecating `PortalAliasID`